app/webpacker/packs/application.js

```diff
@@ -1,9 +1,11 @@
-require.context('govuk-frontend/govuk/assets');
+require.context("govuk-frontend/govuk/assets");
+require("packs/back_link");
```

Comment:
Can we put these in a directory other than 'packs'? Packs are entry points, where as this is just regular code; it would make sense to have it in a different directory, maybe `app/webpacker/scripts`

Reply:
ok